copter/source/docs/ac2_guidedmode.rst

```diff
@@ -67,6 +67,20 @@ Instructions
    modes as "Guided". This may not be the case for other Ground Control
    Stations.
 
+Speed Control
+=============
+
+The maximum horizontal speed of the copter can be adjusted with the
+**Speed** (:ref:`WPNAV_SPEED<WPNAV_SPEED>`) parameter from the Mission Planner's
+Config/Tuning >> Copter Pids screen (see blue box above).  The default
+is 500 meaning 5m/s.  A typical copter can reach top speeds of 10m/s ~
```

Comment:
Thanks for this.  It looks great with one exception.  The WPNAV_SPEED default is actually 1000 (10m/s).  It was 500 a few versions ago but we've increased it now.

Reply:
fixed.....also checked the entire wiki for this default change....only here